instruqt-tracks/terraform-cloud-azure-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -90,28 +90,28 @@ STATUS=$(curl \
     -w "%{response_code}" \
     -s \
     -o /tmp/.out.json \
-    https://app.terraform.io/api/v2/organizations/"${ORG}"/entitlement-set 2>/dev/null) # originally TFC_ORG
+    https://app.terraform.io/api/v2/organizations/"${ORG}"/subscription 2>/dev/null) # originally TFC_ORG
 
 if [ "${STATUS}" != "200" ]; then
-    echo "Failed to get organization entitlements, status ${STATUS}"
+    echo "Failed to get organization subscription, status ${STATUS}"
     if [ -f /tmp/.out.json ]; then
 	echo "Output was:"
 	cat /tmp/.out.json
 	echo "End of output"
     fi
-    fail-message "Unable to get TFC Organization Entitlements, please go back to the \"2- Terraform Cloud Trial Plan\" steps"
+    fail-message "Unable to get TFC Organization Subscription, please go back to the \"2- Terraform Cloud Trial Plan\" steps"
     exit 1
 fi
 
-TRIAL=$(jq -r '.data.attributes."versioned-policy-set-limit"' /tmp/.out.json 2>/dev/null)
+TRIAL=$(jq -r '.included[].attributes."identifier"' /tmp/.out.json 2>/dev/null)
 if [ -z "${TRIAL}" ]; then
-    echo ".data.attributes.versioned-policy-set-limit was empty"
+    echo ".included[].attributes."identifier" was empty"
     fail-message "Unable to determine TFC Trial Status, please go back to the \"2- Terraform Cloud Trial Plan\" steps"
     exit 1
 fi
 
-if [ "${TRIAL}" != "null" ]; then
-    echo ".data.attributes.versioned-policy-set-limit was '${TRIAL}' not 'true'"
+if [ "${TRIAL}" != "trial" ]; then
+    echo ".included[].attributes."identifier" was '${TRIAL}' not 'true'"
```

Comment:
The value should be 'trial'

```suggestion
    echo ".included[].attributes."identifier" was '${TRIAL}' not 'trial'"
```

Reply:
Have to do the inner double-quotes escape here too:

`cho ".included[].attributes.\"identifier\" was '${TRIAL}' not 'trial'"`